src/ape/api/accounts.py

```diff
@@ -156,21 +166,19 @@ def transfer(
             sender=self.address, receiver=receiver, **kwargs
         )
 
-        if value and "send_everything" in kwargs and kwargs["send_everything"]:
-            raise AccountsError("Cannot use 'send_everything=True' with 'VALUE'.")
-
         if data:
             txn.data = self.conversion_manager.convert(data, bytes)
 
-        if value:
-            txn.value = self.conversion_manager.convert(value, int)
-            return self.call(txn)
-
-        elif not kwargs.get("send_everything"):
+        if not value and not kwargs.get("send_everything"):
             raise AccountsError("Must provide 'VALUE' or use 'send_everything=True'")
 
-        else:
-            return self.call(txn, send_everything=True)
+        elif value and "send_everything" in kwargs and kwargs["send_everything"]:
```

Comment:
I am pretty sure this works but I could be missing something dumb:

```suggestion
        elif value and kwargs.get("send_everything"):
```

Reply:
Nice, thanks!